drivers/OpenMote-CC2538/sht21.c

```diff
@@ -131,11 +134,10 @@ uint16_t sht21_read_humidity(void)
     return humidity;

```

Comment:
In addition, the last 2 status bits of the LS Byte (according to the datasheet page 8) should be set to 0 in any case.